package.json

```diff
@@ -8,7 +8,10 @@
     "build:codecs": "npm run build:mozjpeg_enc",
     "start": "webpack serve --host 0.0.0.0 --hot",
     "build": "webpack -p",
-    "lint": "eslint src"
+    "lint": "eslint src",
```

Comment:
Can you update this from `master` to use `tslint`?

Reply:
done!